asv_bench/benchmarks/io/json.py

```diff
@@ -4,7 +4,6 @@
 
 from ..pandas_vb_common import BaseIO
 
-
```

Comment:
Can you undo this please, besides probably being the reason to have the CI in green, we are strict about not including unrelated changes to PR, to keep the maintenance of the project under control.